src/main/scala/elevate/rise/rules/lowering.scala

```diff
@@ -74,4 +74,14 @@ object lowering {
     }
     override def toString = s"slideSeq($rot, $write_dt1)"
   }
+
+  case object toMemAfterMapSeq extends Strategy[Rise] {
```

Comment:
why specifically after mapSeq? in the long run rewriting will deal with read/write annotations, but currently `toMem` is just an identity function that you should be able to put anywhere.

Reply:
We should not be able to put it anyway, only after `write` expressions. But we do not know yet how to determine what a `write` expression is in RISE, or do we?
So this is fixing the test that we fixed together in a slightly more correct way.

Reply:
I will need a rule like this for my camera pipeline, am I going to create specific rules for every `toMem` + write pattern combination? it seems like this rule should remain flexible until we know how to improve it with read/write knowledge.